content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,305 @@
+
+4. केवल काटाकोडा वातावरण के लिए: `30000` टाइप करें, और फिर **Display Port** क्लिक करें।
+
+{{< note >}}
+`dashboard` कमांड डैशबोर्ड ऐड-ऑन को इस्तेमाल के लिए तैयार करता है और प्रॉक्सी को डिफ़ॉल्ट वेब ब्राउज़र में खोलता है।
+आप डैशबोर्ड पर कुबेरनेट्स संसाधन जैसे डेप्लॉयमेंट और सर्विस बना सकते हैं।
+
+यदि आप रुट(root) के रूप में इस्तेमाल कर रहे हैं, तो [URL से डैशबोर्ड खोलना](#open-dashboard-with-url) देखें।
+
+आमतौर पर, डैशबोर्ड केवल आंतरिक कुबेरनेट्स वर्चुअल नेटवर्क के भीतर से ही पहुँचा जा सकता है।
+डैशबोर्ड को कुबेरनेट्स वर्चुअल नेटवर्क के बाहर से एक्सेस करने योग्य बनाने के लिए `dashboard` कमांड एक अस्थायी प्रॉक्सी बनाता है।
+
+प्रॉक्सी को रोकने और प्रक्रिया से बाहर निकलने के लिए `Ctrl+C` का प्रयोग करें।
+कमांड से बाहर निकलने के बाद, डैशबोर्ड कुबेरनेट्स क्लस्टर में चलता रहता है। 
+आप डैशबोर्ड तक पहुंचने के लिए एक और प्रॉक्सी बनाने के लिए फिर से `dashboard` कमांड चला सकते हैं।
+{{< /note >}}
+
+## URL से डैशबोर्ड खोलें {#open-dashboard-with-url}
+
+यदि आप वेब ब्राउज़र नहीं खोलना चाहते हैं, तो URL प्राप्त करने के लिए url फ़्लैग के साथ `dashboard` कमांड चलाएँ:
+
+```shell
+minikube dashboard --url
+```
+
+## डेप्लॉयमेंट बनाएँ
+
+कुबेरनेट्स [*पॉड*](/hi/docs/concepts/workloads/pods/) एक या अधिक कंटेनरों का एक समूह है, 
+जो प्रशासन और नेटवर्किंग के उद्देश्यों के लिए एक साथ बंधे होते हैं। इस ट्यूटोरियल के 
+पॉड में केवल एक कंटेनर है। कुबेरनेट्स 
+[*डेप्लॉयमेंट*](/hi/docs/concepts/workloads/controllers/deployment/) आपके पॉड के स्वास्थ्य की 
+जाँच करता है और यदि पॉड बंद हो जाता है तो पॉड के कंटेनर को पुनः आरंभ करता है।
+पॉड्स के निर्माण और स्केलिंग को प्रबंधित करने के लिए डेप्लॉयमेंट अनुशंसित तरीका है।
+
+1. पॉड को प्रबंधित करने वाला डेप्लॉयमेंट बनाने के लिए `kubectl create` कमांड का उपयोग करें। पॉड 
+प्रदान की गई डॉकर इमेज के आधार पर एक कंटेनर चलाता है।
+
+    ```shell
+    kubectl create deployment hello-node --image=k8s.gcr.io/echoserver:1.4
+    ```
+
+2. डेप्लॉयमेंट देखें:
+
+    ```shell
+    kubectl get deployments
+    ```
+
+    आउटपुट कुछ इस समान होगा:
+
+    ```
+    NAME         READY   UP-TO-DATE   AVAILABLE   AGE
+    hello-node   1/1     1            1           1m
+    ```
+
+3. पॉड देखें:
+
+    ```shell
+    kubectl get pods
+    ```
+
+    आउटपुट कुछ इस समान होगा:
+
+    ```
+    NAME                          READY     STATUS    RESTARTS   AGE
+    hello-node-5f76cf6ccf-br9b5   1/1       Running   0          1m
+    ```
+
+4. क्लस्टर इवेंट देखें:
+
+    ```shell
+    kubectl get events
+    ```
+
+5. `kubectl` कॉन्फ़िगरेशन देखें:
+
+    ```shell
+    kubectl config view
+    ```
+
+{{< note >}}
+`kubectl` कमांड के बारे में अधिक जानकारी के लिए [kubectl अवलोकन](/hi/docs/reference/kubectl/overview/) देखें।
+{{< /note >}}
+
+## सर्विस बनाएं
+
+आमतौर पर, पॉड कुबेरनेट्स क्लस्टर के भीतर अपने आंतरिक IP पते से ही पहुँचा जा सकता है।
+`hello-node` कंटेनर को कुबेरनेट्स वर्चुअल नेटवर्क के
+बाहर से सुलभ बनाने के लिए,पॉड को
+कुबेरनेट्स [*Service*](/hi/docs/concepts/services-networking/service/)(सर्विस) के रूप में बेनकाब करना होगा।
```

Comment:
Has `/docs/concepts/services-networking/service/` been localized into Hindi? If not, don't link there (yet).